contrib/seekable_format/zstdseek_decompress.c

```diff
@@ -23,13 +23,64 @@
 #  endif
 #endif
 
+/* ************************************************************
```

Comment:
Is this change intended? It isn't mentioned in the summary.

Reply:
Yes, 
`PLATFORM_POSIX_VERSION` is used a bit below
but it's not defined anywhere,
so it's silently `== 0` by default, always,
or with the right set of compiler flags, it generates a warning, or an error.

Hence, I define it here, using the same definition as in `platform.h`.

Reply:
Weird... I've been refreshing this page, but GitHub didn't show me this comment until now.

Can you also mention this in the PR summary please? Then the PR LGTM.

Reply:
done